apps/deploy-web/tests/ui/pages/FrontPage.tsx

```
@@ -0,0 +1,29 @@
+import type { BrowserContext as Context, Page } from "@playwright/test";
+import { expect } from "@playwright/test";
+
+import { testEnvConfig } from "../fixture/test-env.config";
+
+export type FeeType = "low" | "medium" | "high";
+export class FrontPage {
+  protected readonly feeType: FeeType = "low";
+
+  constructor(
+    readonly context: Context = context,
```

Biome (2.1.2)

[error] This parameter is used before its declaration.

The parameter is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)